### PR TITLE
wondershaper: add page

### LINKED ISSUE #8006: page request: wondershaper
https://github.com/magnific0/wondershaper

### PR DESCRIPTION
**Pull request by panilya**
Fixes #8006

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/wondershaper.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# wondershaper
+
+> Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.
+> More information: <https://github.com/magnific0/wondershaper#usage>.
+
+- Display help:
+
+`wondershaper -h`
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:8]:
```suggestion
- Print the [h]elp:

`wondershaper -h`
```

File: pages/common/wondershaper.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Show the current status of adapter:
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:10]:
```suggestion
- Show the current [s]tatus of [a]dapter:
```

File: pages/common/wondershaper.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -s -a {{adapter_name}}`
+
+- Clear the limits from adapter:
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:14]:
```suggestion
- Clear the limits from [a]dapter:
```

File: pages/common/wondershaper.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -c -a {{adapter_name}}`
+
+- Set maximum download rate (in Kbps):
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:18]:
```suggestion
- Set maximum [d]ownload rate (in Kbps):
```

File: pages/common/wondershaper.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -a {{adapter_name}} -d 1024`
+
+- Set maximum upload rate (in Kbps):
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:22]:
```suggestion
- Set maximum [u]pload rate (in Kbps):
```

File: pages/common/wondershaper.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -a {{adapter_name}} -u 512`
+
+- Set both maximum download rate and upload rate (in Kpbs):
```
Comment (EmilySeville7cfg) [inline pages/common/wondershaper.md:26]:
```suggestion
- Set both maximum [d]ownload rate and [u]pload rate (in Kpbs):
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/wondershaper.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:8]: It's better to highlight the first character of the short option in the command description.

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/wondershaper.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Print the [h]elp:
+
+`wondershaper -h`
+
+- Show the current [s]tatus of [a]dapter:
+
+`wondershaper -s -a {{adapter_name}}`
+
+- Clear the limits from [a]dapter:
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:14]:
```suggestion
- Clear the limits from an [a]dapter:
```

File: pages/common/wondershaper.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# wondershaper
+
+> Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.
+> More information: <https://github.com/magnific0/wondershaper#usage>.
+
+- Print the [h]elp:
+
+`wondershaper -h`
+
+- Show the current [s]tatus of [a]dapter:
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:10]:
```suggestion
- Show the current [s]tatus of an [a]dapter:
```

File: pages/common/wondershaper.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -s -a {{adapter_name}}`
+
+- Clear the limits from [a]dapter:
+
+`wondershaper -c -a {{adapter_name}}`
+
+- Set maximum [d]ownload rate (in Kbps):
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:18]:
```suggestion
- Set a maximum [d]ownload rate (in Kbps):
```

File: pages/common/wondershaper.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -a {{adapter_name}} -d 1024`
+
+- Set maximum [u]pload rate (in Kbps):
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:22]:
```suggestion
- Set a maximum [u]pload rate (in Kbps):
```

File: pages/common/wondershaper.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -a {{adapter_name}} -u 512`
+
+- Set both maximum [d]ownload rate and [u]pload rate (in Kpbs):
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:26]:
```suggestion
- Set both a maximum [d]ownload rate and a [u]pload rate (in Kpbs):
```

File: pages/common/wondershaper.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -a {{adapter_name}} -d 1024 -u 512`
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:28]:
```suggestion
`wondershaper -a {{adapter_name}} -d {{1024}} -u {{512}}`
```

File: pages/common/wondershaper.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -c -a {{adapter_name}}`
+
+- Set maximum [d]ownload rate (in Kbps):
+
+`wondershaper -a {{adapter_name}} -d 1024`
+
+- Set maximum [u]pload rate (in Kbps):
+
+`wondershaper -a {{adapter_name}} -u 512`
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:24]:
```suggestion
`wondershaper -a {{adapter_name}} -u {{512}}`
```

File: pages/common/wondershaper.md [hunk 14]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -s -a {{adapter_name}}`
+
+- Clear the limits from [a]dapter:
+
+`wondershaper -c -a {{adapter_name}}`
+
+- Set maximum [d]ownload rate (in Kbps):
+
+`wondershaper -a {{adapter_name}} -d 1024`
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:20]:
```suggestion
`wondershaper -a {{adapter_name}} -d {{1024}}`
```

File: pages/common/wondershaper.md [hunk 15]
```diff
@@ -0,0 +1,28 @@
+# wondershaper
+
+> Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:3]:
```suggestion
> Allows the user to limit the bandwidth of one or more network adapters.
```

File: pages/common/wondershaper.md [hunk 16]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://github.com/magnific0/wondershaper#usage>.
+
+- Print the [h]elp:
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:6]:
```suggestion
- Display help:
```

File: pages/common/wondershaper.md [hunk 17]
```diff
@@ -0,0 +1,28 @@
...
+
+`wondershaper -h`
```
Comment (CleanMachine1) [inline pages/common/wondershaper.md:8]: Does this have long versions of these flags such as --help instead?

File: pages/common/wondershaper.md (hunk 8 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:10]: Maybe better:

```suggestion
- Show the current [s]tatus of a specific [a]dapter:
```

File: pages/common/wondershaper.md (hunk 7 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:14]: The same thing here:

```suggestion
- Clear limits from a specific [a]dapter:
```

File: pages/common/wondershaper.md (hunk 9 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:18]:
```suggestion
- Set a specific maximum [d]ownload rate (in Kbps):
```

File: pages/common/wondershaper.md (hunk 10 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:22]:
```suggestion
- Set a specific maximum [u]pload rate (in Kbps):
```

File: pages/common/wondershaper.md (hunk 11 above)
Comment (EmilySeville7cfg) [reply pages/common/wondershaper.md:26]:
```suggestion
- Set a specific maximum [d]ownload rate and [u]pload rate (in Kpbs):
```

I am not sure just about `a` here: is it right to use it when we are talking about several things?

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
LGTM, but a few suggestions 😄

File: pages/common/wondershaper.md (hunk 17 above)
Comment (panilya) [reply pages/common/wondershaper.md:8]: no, it doesn't have

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by blueskyson — COMMENTED** [review]
Look good to me. Thanks for contribution!

**Review by blueskyson — APPROVED** [review]

**Pull request merged**